react-components/package.json

```diff
@@ -14,7 +14,7 @@
     "dev:build-library": "cross-env NODE_ENV=development webpack",
     "test": "jest",
     "test:watch": "jest --watch",
-    "copy-files": "cp ./dist/react-components.js ../rails/app/assets/javascripts/ && cp ./dist/react-components.css ../rails/app/assets/stylesheets/ && cp ./dist/react-test-globals.js ../rails/app/assets/javascripts/ ",
+    "copy-files": "cp ./dist/react-components.js ../rails/app/assets/javascripts/ && cp ./dist/react-test-globals.js ../rails/app/assets/javascripts/",
```

Comment:
Is there no more CSS built by react-components? Or is the CSS now bundled in the javascript?

Reply:
It is all bundled in the Javascript now.  If we don't then we get into issues with webpack exporting some of the css images with md5 hashes which then need to be copied into the rails assets folder.  I thought (for now) it would be simpler to bundle all of it.